src/common/TrackedOp.h

```diff
@@ -181,16 +181,24 @@ class TrackedOp {
   }
   // This function maybe needs some work; assumes last event is completion time
   double get_duration() const {
+    Mutex::Locker l(lock);
     return events.empty() ?
       0.0 :
       (events.rbegin()->first - get_initiated());
   }
 
   void mark_event(const string &event);
   virtual const char *state_string() const {
+    Mutex::Locker l(lock);
     return events.rbegin()->second.c_str();
   }
   void dump(utime_t now, Formatter *f) const;
+  void tracking_start() {
```

Comment:
This code use to be part of the constructor and now occurs immediately after.  It never held the TrackedOp Mutex lock and is identical to master.  There was a lock to protect the old check of tracking_enabled which moved into register_inflight_op().  That doesn't have a lock in hammer because the value can't change.